packages/@react-spectrum/tabs/docs/Tabs.mdx

```diff
@@ -80,21 +80,22 @@ function Example() {
     {id: 2, name: 'Monarchy and Republic', children: 'Senatus Populusque Romanus.'},
     {id: 3, name: 'Empire', children: 'Alea jacta est.'}
   ];
-  let [tabId, setTabId] = React.useState(1);
+  type Tab = typeof tabs[0];
+  let [tabId, setTabId] = React.useState<React.Key>(1);
 
   return (
     <>
       <p>Current tab id: {tabId}</p>
       <Tabs aria-label="History of Ancient Rome" items={tabs} onSelectionChange={setTabId}>
         <TabList>
-          {item => (
+          {(item: Tab) => (
```

Comment:
These are annoying. Because we designed the API with `items`  on `Tabs` rather than `TabList` and `TabPanels`, this will always be `unknown` by default without an annotation.